packages/graphql-language-service/src/interface/__tests__/getAutocompleteSuggestions-test.ts

```diff
@@ -423,6 +425,45 @@ describe('getAutocompleteSuggestions', () => {
         { label: 'String', documentation: GraphQLString.description },
       ]);
     });
+    
+    it('provides correct input type suggestions for fragments', () => {
+      const result = testSuggestions(
+        'fragment someFragment($exampleVariable: ) on Type { ',
+        new Position(0, 41),
+      );
+      expect(result).toEqual([
+        ...metaArgs,
+        { label: 'Boolean', documentation: GraphQLBoolean.description },
+        { label: 'Episode' },
+        { label: 'InputType' },
+        { label: 'Int', documentation: GraphQLInt.description },
+        { label: 'String', documentation: GraphQLString.description },
+      ]);
+    });
+    
+    it.skip('provides correct argument suggestions for fragment-spreads', () => {
+      const externalFragments = parse(`
+        fragment CharacterDetails($someVariable: Int) on Human {
+          name
+        }
+      `, {experimentalFragmentVariables:true}).definitions as FragmentDefinitionNode[];
+
+      const result = testSuggestions(
+        'query { human(id: "1") { ...CharacterDetails() }}',
+        new Position(0, 46),
+        externalFragments,
+      );
+
+      expect(result).toEqual([
+        {
+          label: 'someVariable',
+          insertText: 'someVariable: ',
+          command: suggestionCommand,
+          insertTextFormat: 2,
```

Comment:
if this test is not working yet, even with your graphql-js changes, it may be because we aren't outputting the arguments in `getTypeInfo`